websites/P/PaperMC/metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "387062216030945281",
+		"name": "Romvnly#6466"
+	},
+	"service": "PaperMC",
+	"description": {
+		"en": "PaperMC improves Minecraft's ecosystem with fast, secure software and an expanding plugin API, providing quick releases and helpful support as the most widely used, performant, and stable software available."
+	},
+	"url": [
+		"papermc.io",
+		"docs.papermc.io",
+		"jd.papermc.io",
+		"hangar.papermc.io",
+		"hangar.papermc.dev",
+		"forums.papermc.io"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/SzuS0AE.png",
+	"thumbnail": "https://i.imgur.com/pPFKTFD.jpeg",
```

Comment:
Higher-res version
```suggestion
	"thumbnail": "https://i.imgur.com/Dlt8uSV.jpg",
```

Reply:
are you sure? this thumbnail is kinda watermarked with "PAPER" in the middle ruining the graphic in general

Reply:
Ah, I didn't see that. Well, there should be a higher-resolution version out there somewhere.

Reply:
Here's a better one
```suggestion
	"thumbnail": "https://i.imgur.com/0TPvY1q.png",
```